### ISSUE TITLE
a fresh start

### ISSUE THREAD

**Issue by karimavine** [finndev/PokeBuddy#1214]
what if i created another account  , would i still get banned sometime ?
because my bot account go banned so they flagged the ip ?

**Comment by Rab89** [comment]
yes it will but i'd still proceed in overloading niantics shitty servers
